assets/stylesheets/base/_typography.scss

```diff
@@ -0,0 +1,96 @@
+//TEXT STYLES
+
+%text-small {
+  font-size: 18px;
+  line-height: 1.777777778;
+}
+
+%text-caps {
+  text-transform: uppercase;
+}
+
+%text-bold {
+  font-weight: 600;
+}
+
+%text-color-accent {
+  color: color(orange);
+}
+
+%font-smoothing {
+  -webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+}
+
+h1, h2, h3, h4 {
+  margin-top: 0;
+}
+
+.text--subheading {
+  @extend %text-small;
+  @extend %text-caps;
+  @extend %text-bold;
+  @extend %text-color-accent;
+}
+
+.text {
+  &.-caps {
+    @extend %text-caps;
```

Comment:
This seems like more work than just having the property `text-transform: uppercase` here. Or having a rule like:

```
.caps {
  text-transform: uppercase;
}
```

Not sure much is gained by using placeholders.